nimare/tests/new_json_format/dset_study1.json

```diff
@@ -0,0 +1,75 @@
+{
+  "Studies": [
+    {
+      "Groups": [
+        {
+          "@id": "gg_id1",
+          "Name": "group1",
+          "N": 12
+        }
+      ],
+      "Conditions": [
+        {
+          "@id": "cc_id1",
+          "Name": "household items"
+        },
+        {
+          "@id": "cc_id2",
+          "Name": "animals"
+        }
+      ],
+      "@id": 9593960,
+      "Analyses": [
+        {
+          "Images": {
+            "Elements": [
+              {
+                "Path": "orig/9593960_1_beta.nii.gz",
+                "Value": "Beta"
+              },
+              {
+                "Path": "orig/9593960_1_se.nii.gz",
+                "Value": "StandardError"
```

Comment:
We might need to break this up into multiple components—for things like mean, sd, etc., the computation is independent of the unit. We could maybe use `Quantity` and `Unit` or something like that. Alternatively, we could just have the controlled vocabulary include a bigger list of compound strings (e.g., `Beta`, `BetaStandardError`, etc.).